xercesc.sh

```diff
@@ -0,0 +1,33 @@
+package: XercesC
+version: v3.1.4
+source: https://github.com/PMunkes/XercesC
+tag: v3.1.4
+env:
+  XERCESC_INST_DIR: "$XERCESCINST"
```

Comment:
As there is no upstream git repository we currently need this, unless you were willing to fork it for alisw. This is also the case for a few of the other packages for which I am working on recipes.
Eventually we will fork these repositories for ShipSoft and I'll update the links then.